src/video/sdl2_v.cpp

```diff
@@ -822,20 +824,27 @@ void VideoDriver_SDL::LoopOnce()
 		DrawMouseCursor();
 	}
 
-	/* End of the critical part. */
-	if (_draw_mutex != nullptr && !HasModalProgress()) {
-		_draw_signal->notify_one();
-	} else {
-		/* Oh, we didn't have threads, then just draw unthreaded */
-		Paint();
+	if (is_draw_tick) {
+		_realtime_tick += cur_ticks - last_cur_ticks;
```

Comment:
_realtime_tick is also used in networking code.
So I think it should always be updated along _cur_ticks, for both is_game_tick and is_draw_tick.

Reply:
I even found out our `_realtime_tick` can drift pretty easily :D